### PR TITLE
🌐 🐛 Fix NL translations

### PR DESCRIPTION
**Pull request by JRH-1997**
Fix missing NL translations,
Don't show autoBuyLevels setting
Add missing colors at buildings

### CODE REVIEW

File: src/modules/extendedCallWindow/i18n/nl_NL.root.json [hunk 1]
```diff
@@ -96,6 +97,18 @@
 		"yellowBorder": {
 			"description": "De generatietijd krijgt een geel kader als de inzet ouder is dan een ingesteld aantal uren. Bij 0 is er geen kader.",
 			"title": "Geel kader"
+		},
+		"Overlay": {
```
Comment (Ron31) [inline src/modules/extendedCallWindow/i18n/nl_NL.root.json:101]: lowercase needed

File: src/i18n/nl_NL.ts [hunk 2]
```diff
@@ -196,6 +196,10 @@ export default {
             credits: 5_000,
             coins: 25,
         },
+        {
+            caption: '',
+            color: '',
+        },
```
Comment (KBOE2) [inline src/i18n/nl_NL.ts:202]: We will have to see wether that works. If not, we will probably need to somehow create a workaround

File: src/modules/missionHelper/i18n/nl_NL.root.json [hunk 3]
```diff
@@ -124,6 +128,14 @@
 				"description": "Toont de kop \"Voertuigen\" boven de voertuigvereisten",
 				"title": "Voertuigen (Titel)"
 			}
+		},
+		"Overlay": {
```
Comment (Ron31) [inline src/modules/missionHelper/i18n/nl_NL.root.json:132]: lowercase needed

File: src/modules/missionHelper/settings.ts [hunk 4]
```diff
@@ -47,11 +47,15 @@ export default (MODULE_ID: string): unknown => ({
...
-    },
+    ...(BUILD_LANG !== 'nl_NL'
+        ? {
+            'multifunctionals.heavy_rescue_vehicles': {
+                type: 'toggle',
+                default: false,
+                dependsOn: '.vehicles.content',
+            },
+        }
+        : null),
```
Comment (KBOE2) [inline src/modules/missionHelper/settings.ts:58]: @Suchty112 please check if that is correct. (Note: will be changed to "all languages except nl_NL")

**Review by KBOE2 — CHANGES_REQUESTED** [review]
1. comment just a note
2. Needs statement of Tobi

**Comment by JRH-1997** [comment]
@Ron31 both overlay changed to lowercase

File: src/modules/missionHelper/settings.ts (hunk 4 above)
Comment (Suchty112) [reply src/modules/missionHelper/settings.ts:58]: Currently Yes. can remain so. NL does not have a HLF, i.e. fire engine and rescue vehicle in one

**Review by Suchty112 — APPROVED** [review]

**Review by KBOE2 — APPROVED** [review]

**Pull request merged**